src/subdomains/supporting/payment/dto/tx-fee-details.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { FeeDto } from './fee.dto';
+
+export interface TxFeeDetails {
+  minFee: number;
+  minVolume: number;
+  fee: FeeDto;
+  feeAmount: number;
+}
```

Comment:
Is a bit weird, to have the feeDto as a field, isn't it? I would do it more like this:
```ts
export interface TxFeeDetails extends FeeDto {
   min: number;
   total: number;
   minVolume: number;
}
```

so in the end (just to visualize it) it would be:
```ts
{
   rate: number;
   fixed: number;
   min: number;
   total: number;
   fees: Fee[];
   payoutRefBonus: boolean;
   minVolume: number;
}
```